src/common/subgraphMapper.ts

```diff
@@ -151,7 +151,19 @@ export const mapOrdersArrayToInterface = (
     throw error;
   }
 };
-
+export const mapOrderArrayToPriceid = (response:any) =>{
+  if (response === null) return undefined;
+  try {
+    return response.orders.map((order: Order) => {
+      return  {
+        priceIds :order.market?.feedId
+      }
+    });
+  } catch (error) {
+    console.log('Error while mapping data', error);
```

Comment:
we dont need this try/catch because we are only printing the error but anyway we are throwing and the error is visible on the implementor size.